README.md

```diff
@@ -1,7 +1,32 @@
 ## Lagom Recipes
 
-This repository contains code samples to help understand how to solve common problems. In general, code in each folder of this repository tries to answer a question of type "How do I xyz?".
+This repository contains code samples to help understand how to solve common problems. In general, code in each folder of this repository tries to answer a question of type "How do I _xyz_ ?". For example: "How do I use RDBMS read-sides with Cassandra write-sides?".
```

Comment:
...to help you understand how to achieve common goals? instead of problems, sounds more positive. :-)